tests/system_tests_two_routers.py

```diff
@@ -439,16 +447,35 @@ def bail(self, error):
 
     def on_link_opened(self, event):
         if event.receiver == self.mgmt_receiver:
+            self.mgmt_receiver_link_opened = True
+        elif event.receiver == self.mgmt_receiver_1:
+            self.mgmt_receiver_1_link_opened = True
+        elif event.receiver == self.mgmt_receiver_2:
+            self.mgmt_receiver_2_link_opened = True
+
```

Comment:
shouldn't this check also ensure that "receiver_to_kill" link is also opened?  That ensures the "connection_to_kill" exists before the testing starts.